source/bson-corpus/binary-vector-subtype.md

```diff
@@ -0,0 +1,94 @@
+# BSON Binary Subtype 9 - Vector
+
+- Status: Pending
+- Minimum Server Version: N/A
+
+______________________________________________________________________
+
+## Abstract
+
+This document describes the addition of a new subtype to the Binary BSON type. This subtype is used for efficient
+storage and retrieval of vectors. Vectors here refer to densely packed arrays of numbers, all of the same type.
+
+## Motivation
+
+These representations correspond to the numeric types supported by popular numerical libraries for vector processing,
+such as NumPy, PyTorch, TensorFlow and Apache Arrow. Storing and retrieving vector data using the same densely packed
+format used by these libraries can result in up to 8x memory savings and orders of magnitude improvement in processing
+efficiency. Without this support, MongoDB will be at a competitive disadvantage to databases that do and our users will
+bear the additional cost of storing and processing vector data.
+
+`*` The early addition of the "Packed Bit" representation was to facilitate partnerships in the expanding market of
```

Comment:
What is `*` supposed to be referring to?

Reply:
I've moved the paragraph with the asterisk directly below the table so it should be easier to follow the reference.